app/models/user.rb

```diff
@@ -0,0 +1,16 @@
+class User < ApplicationRecord
+  has_many :votes
```

Comment:
This is where you can put the restriction for limiting how many times a user can vote on works.